Vagrantfile

```diff
@@ -195,10 +195,12 @@ Vagrant.configure("2") do |config|
       end
 
       ip = "#{$subnet}.#{i+100}"
-      node.vm.network :private_network, ip: ip
-      if not Vagrant::Util::Platform.windows? then
-        node.vm.network :private_network, ip: "#{$subnet_ipv6}::#{i+100}"
-      end
+      node.vm.network :private_network, ip: ip,
+        :libvirt__guest_ipv6 => 'yes',
+        :libvirt__ipv6_address => "#{$subnet_ipv6}::#{i+100}",
+        :libvirt__ipv6_prefix => "64",
+        :libvirt__forward_more => "none",
+        :libvirt__dhcp_enabled => false
```

Comment:
you are modifying changes introduced 2 commits ago, squash them

Reply:
Squashed